src/main/java/BlackJackApplication.java

```diff
@@ -0,0 +1,52 @@
+import domain.*;
+import view.InputView;
+import view.OutputView;
+
+public class BlackJackApplication {
+    public static void main(String[] args) {
+        Dealer dealer = new Dealer();
+        Players players = new Players(InputView.inputNames());
+        CardDeck cardDeck = new CardDeck();
+        try {
+            distributeFirstCards(dealer, players, cardDeck);
+            drawMoreCards(dealer, players, cardDeck);
+            printCalculatedResult(dealer, players);
+        } catch (IllegalArgumentException e) {
+            System.out.println(e.getMessage());
+        }
+    }
+
+    private static void printCalculatedResult(Dealer dealer, Players players) {
+        for (Player player : players.getPlayers()) {
+            WinningResult winningResult = player.calculateWinningResult(dealer.getCards().getScore());
```

Comment:
```suggestion
            WinningResult winningResult = player.calculateWinningResult(dealer.getScore());
```

와 같이 사용할 수 있도록 수정하면 좋을 것 같아요.
내부의 값을 공개하는 것 보다는 필요한 기능을 해당 객체에게 역할 위임을 한다고 생각해보세요!

Reply:
막연히 출력할 때는 괜찮다고 생각했는데, 아니었군요! 반영하여 수정하였습니다. 감사합니다 :)